javascript-light-bulb/main.js

```diff
@@ -0,0 +1,15 @@
+const $button = document.querySelector('.yellowButton');
+console.log($button);
```

Comment:
Remove all `console.log` before submitting! They are only for testing / debugging purposes. Unless we specifically ask for them!